tests/braille-specs/afr-za-g2.yaml

```diff
@@ -1136,16 +1136,14 @@ tests:
   - [alfabeties, ⠫⠋⠁⠃⠑⠞⠹⠎]
   - [algaande, ⠫⠛⠣⠙⠑]
   - [algar, ⠫⠛⠁⠗]
-  # FIXME: back translation of words beginning with "alge" which
-  # contract to $= which is the UEB circle shape indicator.
-  - ["alge", "⠫⠿", xfail: true]
-  - [algebra, "⠫⠿⠃⠗⠁", xfail: true]
-  - [algebraïese, ⠫⠿⠃⠗⠁⠘⠒⠊⠑⠎⠑, xfail: true]
-  - [algeheel, ⠫⠿⠓⠱⠇, xfail: true]
-  - [algehele, ⠫⠿⠓⠑⠇⠑, xfail: true]
-  - [algemeen, ⠫⠿⠍⠱⠝, xfail: true]
-  - [algemeenste, ⠫⠿⠍⠱⠝⠌⠑, xfail: true]
-  - [algemene, ⠫⠿⠍⠑⠝⠑, xfail: true]
+#  - ["alge", "⠫⠿"]
+#  - [algebra, "⠫⠿⠃⠗⠁"]
+#  - [algebraïese, ⠫⠿⠃⠗⠁⠘⠒⠊⠑⠎⠑]
+#  - [algeheel, ⠫⠿⠓⠱⠇]
+#  - [algehele, ⠫⠿⠓⠑⠇⠑]
+#  - [algemeen, ⠫⠿⠍⠱⠝]
+#  - [algemeenste, ⠫⠿⠍⠱⠝⠌⠑]
+#  - [algemene, ⠫⠿⠍⠑⠝⠑]
```

Comment:
Why are these tests just commented out (as opposed to set to `xfail` as before)

Reply:
Oops I thought I reverted that.